src/Exceptions/ApiException.php

```diff
@@ -85,4 +87,9 @@ private function getErrorLinkFromHttpBody(): ?string
 
         return null;
     }
+
+    public static function rethrowWithHint(\Exception $e, string $methodName)
+    {
+        return new \Exception(str_replace('_METHOD_', $methodName, self::HINT_MESSAGE), 0, $e);
```

Comment:
Instead of `str_replace` you should use `sprintf()` with just a `%s` placeholder.